### PR TITLE
Add user documentation for ingest/extract formats

### PR DESCRIPTION
**Pull request by charleso**
@blever Sorry I never created the PR (and then forgot). I wasn't very happy with it at the time. Maybe if you have 5 minutes today to talk about it I'm happy to fix up and merge.

### CODE REVIEW

File: doc/user/formats.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+Extract
+-------
+
+These are the formats that can be specified for both `snapshot`/`chord`.
+
+Format                    | Type   | Example                                           | Description
+------------------------- | ------ | ------------------------------------------------- | ------------
+`dense:[ESCAPING:]DELIM`  | text   | `entity|feature1|NA|feature3`                     | All features/values for a single fact are represented in a single line, in which the order is defined in an accompanying `.dictionary`.
+`sparse:[ESCAPING:]DELIM` | text   | `entity|ns|attribute|value|YYYY-MM-DD'T'HH:MM:SS` | Each fact is represented by a separate entry
+`dense:thrift`            | thrift |                                                   | Similar in concept to `dense` text, all values for a single entity are contained in a `List` of values in the same order as the included dictionary.
```
Comment (blever) [inline doc/user/formats.md]: Can we have a pointer to the thrift schema file for this?

File: doc/user/formats.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+`sparse:thrift`           | thrift |                                                   | Similar in concept to `sparse` text, all values for a single entity are contains in a `Map`, where missing values are not included.
```
Comment (blever) [inline doc/user/formats.md]: Same here - a pointer the thrift schema file.

Comment (markhibberd) [inline doc/user/formats.md]: It depends on release, so can it be pointed to in the tarball.

**Comment by etorreborre** [comment]
@charleso it'd be nice as well to specify in the README what is currently in the code (and give a concrete example of the expected paths): 
```scala
If NAMESPACE is set then the input path is expected to contain partitioned factsets
```

**Comment by blever** [comment]
:cherries: if we can point to which thrift schema in the tarball to use for extractions

**Comment by nhibberd** [comment]
@charleso bump

**Pull request merged**